examples/readenvfile.cf

```diff
@@ -0,0 +1,95 @@
+#  Copyright (C) Cfengine AS
+
+#  This file is part of Cfengine 3 - written and maintained by Cfengine AS.
+
+#  This program is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License as published by the
+#  Free Software Foundation; version 3.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+# To the extent this program is licensed as part of the Enterprise
+# versions of Cfengine, the applicable Commercial Open Source License
+# (COSL) may apply to this file if you as a licensee so wish it. See
+# included file COSL.txt.
+
+#+begin_src prep
+#@ ```
+#@ echo 'PRETTY_NAME="Ubuntu 14.04.5 LTS"' > /tmp/os-release
+#@ touch /tmp/os-release.json
+#@ ```
+#+end_src
+###############################################################################
+#+begin_src cfengine3
+
+body edit_defaults empty
+{
+      empty_file_before_editing => "true";
+      edit_backup => "false";
+}
+
+bundle edit_line insert_lines(lines)
+{
+    insert_lines:
+        "$(lines)";
+}
+
+body printfile cat(file)
+{
+        file_to_print => "$(file)";
+        number_of_lines => "inf";
+}
+
+# The above bodies can be imported from the standard library (if installed):
+# body file control { inputs => { "$(sys.libdir)/stdlib.cf" }; }
+
+body common control
+{
+      bundlesequence => { "readenvfile_example" };
+}
+
```

Comment:
Does it still work if the src begin is moved to here? then the rendered example would hide the stuff from the stdlib